src/common/SplitView/SplitViewSized.tsx

```diff
@@ -38,6 +38,7 @@ const SplitViewSized = ({ children }: SizedPaneProps) => {
   return (
     <Box
       pointerEvents={dragging ? "none" : "unset"}
+      display={mode === "collapsed" ? "none" : undefined}
       visibility={mode === "collapsed" ? "hidden" : undefined}
```

Comment:
This fixes some of the extra space around the UI that is added when the simulator is collapsed. `visibility: hidden` does not change the layout of the document, `display: none` does.